modules/retail/pages/retail-terminal-names.adoc

```diff
@@ -14,5 +30,115 @@ For example:
+
+
+
+=== Assign Hostnames to Terminals
+
+
+If you want terminal names to be derived from the hostname, you will need to ensure your terminals have a static hostname.
+This requires a static IP address to be able to resolve the static hostname.
+
+There are a number of different ways to assign hostnames to terminals.
+This section describes how to do this when DNS and DHCP services are managed by the branch server.
+
+
+.Procedure: Assigning IP Address and Hostname with Formulas
+
+. In the DHCP formula settings, navigate to [guimenu]``Hosts with Static IP Address`` and click btn:[Add Item].
+For more information on the DHCP formula, see xref:salt:formula-dhcpd.adoc[].
+. In the [guimenu]``Hostname`` field, type the hostname of the branch server.
+. In the [guimenu]``IP Address`` field, type the static IP address for the terminal.
+Ensure the IP address is within the range used by the branch server.
+. In the [guimenu]``Hardware Type and Address`` field, type the hardware type and address in this format:
++
+----
+ethernet <terminal_MAC_address>
+----
+. OPTIONAL: For multiple terminals, click btn:[Add Item] and fill in the details for each terminal.
+. Click btn:[Save Formula] to save the changes.
+. In the Bind formula settings, navigate to the A records of the appropriate non-reverse zone, and click btn:[Add Item].
+For more information on the bind formula, see xref:salt:formula-bind.adoc[].
+. In the [guimenu]``Hostname`` field, type the hostname of the branch server.
+. In the [guimenu]``IP Address`` field, type the static IP address you assigned to the terminal in the DHCP formula settings.
+. OPTIONAL: For multiple terminals, click btn:[Add Item] and fill in the details for each terminal.
+. Click btn:[Save Formula] to save the changes.
+. Apply the highstate on the branch server to apply the changes, and update the terminal with the new hostname.
+
+[[IMPORTANT]]
+====
+If the terminal was previously registered using a name based on the hardware type instead of the hostname, you will need to delete the original registration.
+When you re-register the terminal, use the new terminal name.
+====
+
+
+
+.Procedure: Assigning IP Address and Hostname with YAML
+
+. At the command prompt on the branch server, export a YAML configuration file:
++
+----
+retail_yaml --to-yaml retail.yaml
+----
+. Open the YAML file and navigate to the end of the branch server section.
+Add a new [systemitem]``terminals`` section if it does not already exist.
+. Add the IP address, MAC address, and hardware type for the terminal, using this format:
++
+----
+$hostname:
+        IP: <IP_Address>
+        hwAddress: <MAC_Address>
+        hwtype: <HWTYPE_Group_name_without_HWTYPE:_prefix>
+----
+. Import the modified YAML file:
++
+----
+retail_yaml --from-yaml retail.yaml
+----
+. Apply the highstate on the branch server to apply the changes, and update the terminal with the new hostname.
```

Comment:
Maybe, the comma is superfluous or we could say something like this:
```suggestion
. Apply the highstate on the branch server to apply the changes, and thus updating the terminal with the new hostname.
```
Otherwise I read it as if "update the terminal with the new hostname" would be a second step.  But my impression could be wrong, misguided by the way we use commas in German.

Reply:
Yeah, that's a serial (Oxford) comma, so the part of the sentence after the comma is not a separate clause. I think removing the comma might make it clearer, but also not en-US ;)

Reply:
Looking at this closer, that last part is a result statement anyway, so deleting it is probably the best thing to do. Let me know what you think.